functions-and-operators/string-functions.md

```diff
@@ -1388,11 +1388,51 @@ Remove leading and trailing spaces.
 
 ### [`UCASE()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_ucase)
 
-Synonym for `UPPER()`.
+The `UCASE()` function is used to convert a string to uppercase letters. This function is equivalent to the `UPPER()` function.
+
+> **Note:**
+>
+> When the string is `NULL`, the `UCASE()` function returns `NULL`.
+
+Example:
+
+```sql
+SELECT upper('bigdata') AS result_upper, upper('null') AS result_null;
+```
+
+Output:
+
+```sql
++--------------+-------------+
+| result_upper | result_null |
++--------------+-------------+
+| BIGDATA      | NULL        |
++--------------+-------------+
+```
 
 ### [`UNHEX()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_unhex)
 
-Return a string containing hex representation of a number.
+The `UNHEX()` function performs the reverse operation of the `HEX()` function. It treats each pair of characters in the argument as a hexadecimal number and converts it to the character represented by that number, returning the result as a binary string.
+
+> **Note:**
+>
+> The argument string must be a valid hexadecimal value that contains `0`~`9`, `A`~`F`, `a`~`f`. If the argument string is `NULL` or falls outside this range, it returns `NULL`.
+
```

Comment:
The GitHub preview looks weird. Let me set up a doc website preview. 

![Uploading image.png…]()

Reply:
The website preview looks OK. 
https://pingcap-docsite-preview-git-pr-qiancai.vercel.app/tidb/dev/string-functions#unhex
![image](https://github.com/pingcap/docs/assets/79440533/7a92bf7b-0751-4c81-96c7-bc294f612377)

Reply:
@qiancai Thanks! I just tried another format using "-", which shows well on the GitHub page.